detections/endpoint/windows_ad_domain_controller_audit_policy_disabled.yml

```diff
@@ -53,7 +53,7 @@ rba:
   threat_objects: []
 tags:
   analytic_story:
-  - Sneaky Active Directory Persistence Tricks
```

Comment:
is there a reason to replace Sneaky? Can we add both the stories?

Reply:
It was removed because "Sneaky" is unrelated to the analytic. The Sneaky story talks about persistence whereas the analytic cares more about audit policies being disabled. Hence the removal